packages/pluggableWidgets/accordion-native/src/components/AccordionGroup.tsx

```diff
@@ -32,21 +32,32 @@ export function AccordionGroup({
     visible,
     style
 }: AccordionGroupProps): ReactElement | null {
+    const isAccessible = group.accessible === "yes";
     return visible ? (
         <View style={style.container}>
             <Pressable
                 style={[style.header.container, icon === "left" && { flexDirection: "row-reverse" }]}
                 onPress={collapsible ? () => onPressGroupHeader(group, index) : null}
+                accessible={isAccessible}
+                accessibilityLabel={
+                    group.screenReaderCaption?.value ||
+                    (group.headerRenderMode === "text" ? group.headerText.value : undefined)
+                }
+                accessibilityHint={group.screenReaderHint?.value}
+                accessibilityState={{ expanded: isExpanded }}
```

Comment:
you can use `accessibilityElementsHidden` and `importantForAccessibility` properties to disable all the children if the parent is disabled